src/Microsoft.ML.Data/Scorers/PredictionTransformer.cs

```diff
@@ -500,13 +517,15 @@ private protected override void SaveCore(ModelSaveContext ctx)
             base.SaveCore(ctx);
 
             ctx.SaveStringOrNull(_trainLabelColumn);
+            ctx.SaveStringOrNull(_scoreColumn);
+            ctx.SaveStringOrNull(_predictedLabelColumn);
         }
 
         private static VersionInfo GetVersionInfo()
         {
             return new VersionInfo(
                 modelSignature: "MC  PRED",
-                verWrittenCur: 0x00010001, // Initial
+                verWrittenCur: 0x00010002, // Initial
```

Comment:
It would be good to ensure we have a test explicitly testing the old format.

If necessary, this can be logged as an issue and added in a follow-up PR.

Reply:
[https://github.com/dotnet/machinelearning/issues/4389](https://github.com/dotnet/machinelearning/issues/4389)

---
In reply to: [339196504](https://github.com/dotnet/machinelearning/pull/4372#discussion_r339196504) [](ancestors = 339196504)